news/2025-05-29-cuda-118.md

```diff
@@ -40,6 +40,31 @@ cuda_compiler_version:
   - 11.8                    # [linux or win]
   - 12.4                    # [linux and ppc64le]
   - 12.8                    # [(linux and not ppc64le) or win]
+docker_image:                                               # [linux]
+  # non-CUDA builds
+  - quay.io/condaforge/linux-anvil-x86_64:alma9             # [linux and os.environ.get("BUILD_PLATFORM") == "linux-64"]
+  - quay.io/condaforge/linux-anvil-aarch64:alma9            # [linux and os.environ.get("BUILD_PLATFORM") == "linux-aarch64"]
+  - quay.io/condaforge/linux-anvil-ppc64le:alma9            # [linux and os.environ.get("BUILD_PLATFORM") == "linux-ppc64le"]
+
+  # CUDA 11.8 builds
+  - quay.io/condaforge/linux-anvil-x86_64-cuda11.8:ubi8     # [linux64 and os.environ.get("BUILD_PLATFORM") == "linux-64"]
+  # CUDA 11.8 arch: native compilation (build == target)
+  - quay.io/condaforge/linux-anvil-aarch64-cuda11.8:ubi8    # [aarch64 and os.environ.get("BUILD_PLATFORM") == "linux-aarch64"]
+  - quay.io/condaforge/linux-anvil-ppc64le-cuda11.8:ubi8    # [ppc64le and os.environ.get("BUILD_PLATFORM") == "linux-ppc64le"]
+  # CUDA 11.8 arch: cross-compilation (build != target)
+  - quay.io/condaforge/linux-anvil-x86_64-cuda11.8:ubi8     # [aarch64 and os.environ.get("BUILD_PLATFORM") == "linux-64"]
+  - quay.io/condaforge/linux-anvil-x86_64-cuda11.8:ubi8     # [ppc64le and os.environ.get("BUILD_PLATFORM") == "linux-64"]
+
+  # CUDA 12.x builds
+  - quay.io/condaforge/linux-anvil-x86_64:alma9             # [linux and os.environ.get("BUILD_PLATFORM") == "linux-64"]
+  - quay.io/condaforge/linux-anvil-aarch64:alma9            # [linux and os.environ.get("BUILD_PLATFORM") == "linux-aarch64"]
+  - quay.io/condaforge/linux-anvil-ppc64le:alma9            # [linux and os.environ.get("BUILD_PLATFORM") == "linux-ppc64le"]
+
+zip_keys:                       # [linux or win]
+    - cuda_compiler             # [linux or win]
+    - cuda_compiler_version     # [linux or win]
+    - docker_image              # [linux]
```

Comment:
To keep this overridable, I think we have to keep the docker_image in the CUDA-zip also on the global pinning. If we reduced the global pinning to
```yaml
zip_keys:                       # [linux or win]
    - cuda_compiler             # [linux or win]
    - cuda_compiler_version     # [linux or win]
```
then feedstocks would be able to squeeze the `docker_image` key into that. It shouldn't be a big burden to keep the docker-image in that (already much reduced zip), so I'm fine to keep it. That would then also make it unnecessary to add the `zip_keys` in the example here.